src/video/kmsdrm/SDL_kmsdrmvideo.c

```diff
@@ -437,9 +481,33 @@ static void KMSDRM_UnlockHWSurface(_THIS, SDL_Surface *surface)
 	return;
 }
 
+static int KMSDRM_FlipHWSurface(_THIS, SDL_Surface *surface)
+{
+	if ( !drm_active_pipe )
+		return -2;
+	
+	if (this->screen->flags & SDL_DOUBLEBUF) {
+		// drmModeObjectSetProperty seems to wait for VBlank already, this doesn't
+		// seem really necessary.
+		/*drmVBlank vb = {
+			.request.type = DRM_VBLANK_RELATIVE,
+			.request.sequence = 1,
+		};
+
+		drmWaitVBlank(drm_fd, &vb);*/
+		drmModeObjectSetProperty(drm_fd, drm_active_pipe->plane, 
```

Comment:
setting "FB_ID" on the plane here is a blocking operation that waits for vblank, is this expected behavior?